src/main/java/frc/robot/RobotContainer.java

```diff
@@ -283,204 +345,232 @@ private void configureButtonBindings() {
-        .whileTrue(intake.humanPlayerIntake());
-    OI.getTrigger(OI.Driver.RTrigger).and(() -> intakeAlgeaMode).whileTrue(intake.algaeIntake());
-    OI.getTrigger(OI.Driver.RTrigger).and(() -> intakeAlgeaMode).whileFalse(intake.algaeHold());
-    Command locateCoral =
-        new LocateCoral(
-            sensors::getSensorState,
-            intake,
-            coralOuttakeButton.or(OI.getTrigger(OI.Driver.LTrigger)));
-
-    intake
-        .intakeHasUnalignedCoralTrigger()
-        .and(coralOuttakeButton.negate())
-        .and(OI.getTrigger(OI.Driver.LTrigger).negate())
-        .and(() -> !CommandScheduler.getInstance().isScheduled(scoreL1))
-        .onTrue(locateCoral);
-
-    OI.getButton(OI.Driver.A)
-        .whileTrue(intake.conveyerInCommand().alongWith(coralScorer.intakeCommand()));
-    // .until(coralHandoffCompleteTrigger));
-
-    intake
-        .intakeHasCoralTrigger()
-        .and(() -> elevatorNotL1)
-        .and(coralOuttakeButton.negate())
-        .and(() -> !CommandScheduler.getInstance().isScheduled(locateCoral))
-        .and(elevator.elevatorAtSetpoint(ElevatorConstants.kL0Height))
-        .onTrue(
-            Robot.isReal()
-                ? intake
-                    .conveyerInCommand()
-                    .alongWith(coralScorer.intakeCommand())
-                    .until(coralHandoffCompleteTrigger)
-                : Commands.runOnce(() -> mapleSimArenaSubsystem.setRobotHasCoral(true)));
-    coralOuttakeButton.whileTrue(intake.floorOuttake());
-
-    Logger.recordOutput("Intake/Modes/L1 Score Mode", !elevatorNotL1);
-    Logger.recordOutput("Intake/Modes/Algae Mode", intakeAlgeaMode);
-    Logger.recordOutput("Intake/Modes/Coral Station Mode", coralStationMode);
-    OI.getButton(OI.Operator.Y)
-        .onTrue(
-            Commands.runOnce(
-                () -> {
-                  elevatorNotL1 = !elevatorNotL1;
-                  Logger.recordOutput("Intake/Modes/L1 Score Mode", !elevatorNotL1);
-                }));
-    OI.getButton(OI.Operator.X)
-        .onTrue(
-            Commands.runOnce(
-                () -> {
-                  intakeAlgeaMode = !intakeAlgeaMode;
-                  Logger.recordOutput("Intake/Modes/Algae Mode", intakeAlgeaMode);
-                }));
-    OI.getButton(OI.Operator.B)
-        .onTrue(
-            Commands.runOnce(
-                () -> {
-                  coralStationMode = !coralStationMode;
-                  Logger.recordOutput("Intake/Modes/Coral Station Mode", coralStationMode);
-                }));
-    OI.getButton(OI.Driver.X).whileTrue(intake.l1ScoreModeB()); // Temporary
-    OI.getTrigger(OI.Driver.LTrigger)
-        .and(() -> !elevatorNotL1 && !intakeAlgeaMode)
-        .whileTrue(scoreL1); // Temporary
-    intake.setDefaultCommand(intake.Idle());
+    if (intake != null
+        && sensors != null
+        && coralScorer != null
+        && elevator != null
+        && mapleSimArenaSubsystem != null) {
```

Comment:
This is too broad for some of these mappings. Many only require the intake, for instance.
mapleSimArenaSubsystem will be null when we aren't simulating, which means this will always get skipped on the real robot.